src/adapter/http/src/axum_utils.rs

```diff
@@ -13,6 +13,21 @@ use kamu_accounts::{AnonymousAccountReason, CurrentAccountSubject};
 use kamu_core::auth;
 use thiserror::Error;
 
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Macros
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+macro_rules! from_catalog_n {
+    ($catalog:ident, $T:ty) => {{
+        $catalog.get_one::<$T>().unwrap()
+    }};
+    ($catalog:ident, $T:ty, $($Ts:ty),+) => {{
+        ( $catalog.get_one::<$T>().unwrap(), $( $catalog.get_one::<$Ts>().unwrap() ),+ )
+    }};
+}
+
+pub(crate) use from_catalog_n;
```

Comment:
I think we had a similar thing in GQL adapter. Does this mean it should be a part of `dill`?